index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Monicah capstone project</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/a6627283f4.js" crossorigin="anonymous"></script>
+    <link href="https://www.dafontfree.net/embed/Y29jb2dvb3NlLXJlZ3VsYXImZGF0YS84L2MvNDU1NjUvQ29jb2dvb3NlX3RyaWFsLm90Zg" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+    <section class="main-page">
+        <nav class="top-bg">
+            <div>
+            
+                <ul class="top">
+                    <a href=""><i class="fa-brands fa-facebook-f"></i></a>
+                    <a href=""><i class="fa-brands fa-twitter"></i></a>
+                  <li>Engish</li>
+                  <li>My page</li>
+                  <li>Logout</li>
+                </ul>
+            </div>
+        </nav>
+        <header class="header" >
+            <div class="lg-logo">
+                <a href="homepage.html">
+                  <img src="Images/compost_FILL.svg" alt="logo img">
+                </a>
+            <a  class="logo" href="">Eco Tourism<br> Kenya 2022 </a>
+            </div>
+            
+            <nav class="nav-1">
+              
+                    
+                
+                <ul class="list">
+
+                    <li class="list-la bar-list"><a>About</a></li>
+                    <li class="bar-list"><a class="go" href="homepage.html">program</a></li>
+                    <li class="bar-list"><a>Join</a></li>
+                    <li class="bar-list"><a>Sponsor</a></li>
+                    <li class="bar-list"><a>News</a></li>
+                    <li class="btn bar-list"><a>CC Captain</a></li>
+                </ul>
```

Comment:
Well done on implemented some of the changes requested by the previous reviewer :+1: I can see you have really put in a lot of effort. However, you need to take your time and make sure everything works effectively. :wink: Kindly ensure that the links work properly, clicking on the `about` should take your to the about page, and the `program` or `logo` should take you to the home page. :1st_place_medal:  This will make your project more appealling and user-friendly. :smile_cat: